app/src/main/java/com/worldsnas/starplayer/view/musics_list/MusicsListFragment.kt

```diff
@@ -127,8 +127,8 @@ class MusicsListFragment : Fragment() {
         }
     }
 
-    private fun musicListener(localMusic: LocalMusic) {
-        val bundle = bundleOf(ConstValues.BUNDLE_KEY_MUSIC_INFO to localMusic)
+    private fun musicListener(music: Music) {
+        val bundle = bundleOf(ConstValues.BUNDLE_KEY_MUSIC_INFO to music)
 
         findNavController()
             .navigate(R.id.action_musicsListFragment_to_playerFragment, bundle)
```

Comment:
Az safe args estefade kind behtare

Reply:
chera , dalil khasi dare?
in kare badi hast?

Reply:
amniat bishtari mide nesbat be code